modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -4,27 +4,22 @@
 [id="gitops-creating-a-new-client-in-dex_{context}"]
 = Enabling the Dex OpenShift OAuth Connector
 
-Dex uses the users and groups defined within OpenShift by checking the `OAuth` server provided by the platform. The following example shows the properties of Dex along with example configurations:
+Dex is installed by default for all the Argo CD instances created by the Operator. You can configure {gitops-title} to use Dex as the SSO authentication provider by setting the `.spec.sso` parameter. 
```

Comment:
> created by the Operator

Do we need to state which Operator?

Reply:
@mburke5678 not required. It is understood as the GitOps Operator.